roles/openshift_node/tasks/main.yml

```diff
@@ -124,3 +124,15 @@
 
 - set_fact:
     node_service_status_changed: "{{ start_result | changed }}"
+    
+- name: Check access to port 1936
+  script: check-1936-access.sh {{ openshift.common.hostname }}
+  register: check1936
+  failed_when: false
+  changed_when: false
+
+- name: Warn about restricted access to port 1936
+  pause:
+    prompt: "Your node {{ ansible_host }} cannot access {{ openshift.common.hostname }} on port 1936, this host will fail default router liveness checks. Press CTRL-C to abort or continue"
+    seconds: 30
+  when: check1936.rc != 0
```

Comment:
I did this because if a router wasn't deployed on the host the check would fail even if the networking rules were fine because nothing was listening so we need to use `nc` to stub out a listener to verify things. Does that make sense?